packages/ripple-keypairs/tsconfig.eslint.json

```diff
@@ -0,0 +1,4 @@
+{
+  "extends": "./tsconfig.json",
+  "include": ["test/**/*.ts"]
```

Comment:
Why does this not include the source files?

Reply:
Mistake on my part, good catch